packages/addon-dev/src/rollup-app-reexports.ts

```diff
@@ -8,6 +8,7 @@ export default function appReexports(opts: {
   to: string;
   include: string[];
   mapFilename?: (filename: string) => string;
+  defaultExport?: string;
```

Comment:
I think we need a better name for this. It's not only about default exports, it's about exports in general.